tests/gravity_point.prm

```diff
@@ -0,0 +1,114 @@
+# A simple setup for gravity acceleration calculated at a point P
+# manual for more information.
```

Comment:
Please include more detailed comments here. Say what post processor you are testing, what the output is supposed to look like (analytical solution?) and what specific features of the gravity post processor you are testing.